web-ui/src/main/webapp/js/npm.js

```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
```

Comment:
What is this and why do you need it?

Reply:
It's in the bootstrap js directory which i downloaded from the offical site